packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -935,36 +935,53 @@ export class Stack extends Construct implements ITaggable {
       return Fn.importValue(options.name);
     }
 
-    const resolvable = Tokenization.reverse(exportedValue);
-    if (!resolvable || !Reference.isReference(resolvable)) {
-      throw new Error('exportValue: either supply \'name\' or make sure to export a resource attribute (like \'bucket.bucketName\')');
+    const importValue = this.determineImportValue(exportedValue);
+
+    if (Array.isArray(importValue)) {
+      throw new Error('Attempted to export a list value from `exportValue()`: use `exportListValue()` instead');
     }
 
-    // "teleport" the value here, in case it comes from a nested stack. This will also
-    // ensure the value is from our own scope.
-    const exportable = getExportable(this, resolvable);
+    return importValue;
+  }
 
-    // Ensure a singleton "Exports" scoping Construct
-    // This mostly exists to trigger LogicalID munging, which would be
-    // disabled if we parented constructs directly under Stack.
-    // Also it nicely prevents likely construct name clashes
-    const exportsScope = getCreateExportsScope(this);
+  /**
+   * Create a CloudFormation Export for a value
+   *
+   * Returns a string list representing the corresponding `Fn.importValue()`
+   * expression for this Export. The export expression is automatically wrapped with an
+   * `Fn::Join` and the import value with an `Fn::Split`, since CloudFormation can only
+   * export strings. You can control the name for the export by passing the `name` option.
+   *
+   * If you don't supply a value for `name`, the value you're exporting must be
+   * a Resource attribute (for example: `bucket.bucketName`) and it will be
+   * given the same name as the automatic cross-stack reference that would be created
+   * if you used the attribute in another Stack.
+   *
+   * One of the uses for this method is to *remove* the relationship between
+   * two Stacks established by automatic cross-stack references. It will
+   * temporarily ensure that the CloudFormation Export still exists while you
+   * remove the reference from the consuming stack. After that, you can remove
+   * the resource and the manual export.
+   *
+   * # See `exportValue` for an example of this process.
+   */
+  public exportListValue(exportedValue: any, options: ExportValueOptions = {}): string[] {
+    if (options.name) {
+      new CfnOutput(this, `Export${options.name}`, {
+        value: exportedValue,
```

Comment:
Doens't this need a `Fn.join()` as well? A direct call to this needs a test as well then.

Do we need `CfnOutput` to check for a list and throw an error?